### PR TITLE
Bump Calamari.Common to 19.4.10 (Server 2021.2 stream)

### PR DESCRIPTION
**Pull request by mjhilton**
# Overview
This PR is one of a set of PRs to propagate a change to Calamari.Common through to Octopus Server 2021.2

Upstream change:
* OctopusDeploy/Calamari#808

To fix Issues:
* OctopusDeploy/Issues#6794
* OctopusDeploy/Issues#7177

### CODE REVIEW

File: source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     <ItemGroup>
         <PackageReference Include="Assent" Version="1.6.1" />
-        <PackageReference Include="Calamari.Common" Version="19.2.5" />
```
Comment (andrewabest) [inline source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj:17]: 😬

**Review by andrewabest — APPROVED** [review]
Version numbers line up for 2021.2, LGTM :)

File: source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj (hunk 1 above)
Comment (mjhilton) [reply source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj:17]: ? Concerned about minor patch? Or just that this reference was left behind for a while?

Comment (andrewabest) [reply source/Calamari.Tests.Shared/Calamari.Tests.Shared.csproj:17]: Yeah the latter, it is nice we are bringing it back in line :)

**Comment by mjhilton** [comment]
Yes, through a very technically advanced version management process
image

😬

**Pull request merged**